templates/partials/topic/post.tpl

```diff
@@ -82,6 +82,14 @@
 
 		<!-- IMPORT partials/topic/post-menu.tpl -->
 	</small>
+
+	<!-- IF posts.replies -->
+	<br />
+	<a component="post/reply-count" href="#" class="no-select">
+		<i class="fa fa-plus"></i>
```

Comment:
Can you add a component to this maybe `[component="post/reply/toggle"]` so we can use selector in core. This will allow themes to use different icons like `fa-chevron-down`